### PR TITLE
IE11 unsupported banner

### PR DESCRIPTION
**Pull request by smusoke**
**What**
A new persistent banner across all of dashboard with copy "We no longer support this browser. Please try visiting this page using Edge, Chrome, Firefox, or another modern browser." for all Internet Explorer browsers

**Why**
IE _will_ be deprecated as a whole. Anything previous to IE 11 is already unsupported.

**Note:** IE11 isn't officially unsupported on the platform _yet_

## Links

- jira ticket: [FND-1615](https://codedotorg.atlassian.net/browse/FND-1615)

## Testing story

Tested locally using [Microsoft provided VMs](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)
1. Tested Chrome
2. Tested IE 11 (on VM)
3. Tested IE 9

## Screenshots
Chrome
Screen Shot 2021-07-29 at 4 46 08 PM

IE11
Screen Shot 2021-07-29 at 4 46 13 PM

IE9 (Unsupported by our platform)
Screen Shot 2021-07-29 at 4 46 18 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -620,11 +620,10 @@ en:
   compatibility:
     upgrade:
       unsupported_message: "Your browser is not supported. Please upgrade your browser to [one of our supported browsers](%{supported_browsers_url}). You can try viewing the page, but expect functionality to be broken."
+      ie_unsupported_message: "We no longer support this browser. Please try visiting this page using Edge, Chrome, Firefox, or another modern browser."
```
Comment (daynew) [inline dashboard/config/locales/en.yml]: Why have the new `ie_unsupported_message` if the `unsupported_message` string already exists?

Comment (smusoke) [reply dashboard/config/locales/en.yml]: The copy change was listed under the [deliverable section in the ticket](https://codedotorg.atlassian.net/browse/FND-1615). Assumed the existing message should continue to exist for all other browsers that just need to be upgraded.

**Review by daynew — APPROVED** [review]
So simple!

**Comment by dju90** [comment]
The banner doesn't show up on IE 9 - is that because that browser can't render it?

**Comment by smusoke** [comment]
> The banner doesn't show up on IE 9 - is that because that browser can't render it?

@dju90 Yeah IE 9 was unsupported to begin with. I wasn't able to even load the page running locally with none of changes in this PR

**Pull request merged**